src/pr-info.ts

```diff
@@ -421,6 +421,10 @@ export function getPackagesTouched(files: readonly FileInfo[]) {
     return [...new Set(noNulls(files.map(f => "package" in f ? f.package : null)))];
 }
 
+export function getPackagesWithSourceFilesTouched(files: readonly FileInfo[]) {
+    return [...new Set(noNulls(files.map(f => "package" in f && f.kind === "definition" ? f.package : null)))];
+}
```

Comment:
Felt pretty reasonable to do, basically the other 

````
type FileKind = "test" | "definition" | "markdown" | "package-meta" | "package-meta-ok"| "infrastructure";

````

is the full list, maybe it could also take into account `package-meta-ok`?

Reply:
It was originally "more typed", maybe it's best to return to some more structured thing, like with interface extensions, so there's a type for everything that has `package` and `suspect` would similarly exist only on `...-meta-ok`.

(It's a small thing though, so I hesitated to make it much bigger just for this.)